### ISSUE TITLE
1.1.0 downloaded Plugins too but 1.2.0 only downloads dependencies

### ISSUE THREAD

**Issue by softwaremaverick** [qaware/go-offline-maven-plugin#10]
I've just noticed whilst using 1.2.0 that I'm now missing an artefact that does appear in 1.1.0.

As far as I can see the other artefacts I have are declared as dependencies, whereas this one is declared as a plugin

Weirdly enough (for some reason) I do end up with the pom files for that plugin in my m2 repo but not the jar.

Would this be easy to resolve? 

Notably, this is after running 1.1.0

```
xliffgen
xliffgen/ers-xliffgen-maven-plugin
xliffgen/ers-xliffgen-maven-plugin/1.2
xliffgen/ers-xliffgen-maven-plugin/1.2/ers-xliffgen-maven-plugin-1.2.pom.sha1
xliffgen/ers-xliffgen-maven-plugin/1.2/ers-xliffgen-maven-plugin-1.2.pom
xliffgen/ers-xliffgen-maven-plugin/1.2/ers-xliffgen-maven-plugin-1.2.jar.sha1
xliffgen/ers-xliffgen-maven-plugin/1.2/_remote.repositories
xliffgen/ers-xliffgen-maven-plugin/1.2/ers-xliffgen-maven-plugin-1.2.jar
```

and this is after running 1.2.0
```
xliffgen
xliffgen/ers-xliffgen-maven-plugin
xliffgen/ers-xliffgen-maven-plugin/1.2
xliffgen/ers-xliffgen-maven-plugin/1.2/ers-xliffgen-maven-plugin-1.2.pom.sha1
xliffgen/ers-xliffgen-maven-plugin/1.2/ers-xliffgen-maven-plugin-1.2.pom
xliffgen/ers-xliffgen-maven-plugin/1.2/_remote.repositories
```

**Comment by softwaremaverick** [comment]
I believe in the meantime I can run
mvn org.apache.maven.plugins:maven-dependency-plugin:3.1.1:resolve-plugins

to resolve them.

**Comment by Theoderich** [comment]
Hi @softwaremaverick 
Thank you for the bugreport. :)
I think I found the problem: 1.2.0 only downloads the transitive dependencies of plugins, but not the plugin artifact itself. I have pushed a commit to master that should fix the problem. Can you build the current master and test if your problem is fixed?

**Comment by softwaremaverick** [comment]
Thanks. I'll try this on Monday when I'm back in the office.

On 8 June 2019 13:55:52 BST, Theoderich <notifications@github.com> wrote:
>Hi @softwaremaverick 
>Thank you for the bugreport. :)
>I think I found the problem: 1.2.0 only downloads the transitive
>dependencies of plugins, but not the plugin artifact itself. I have
>pushed a commit to master that should fix the problem. Can you build
>the current master and test if your problem is fixed?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/qaware/go-offline-maven-plugin/issues/10#issuecomment-500122173

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by Theoderich** [comment]
I have released version 1.2.1 that should fix the problem
